packages/api/src/webhooks/cannon.ts

```diff
@@ -75,7 +82,8 @@ export default class WebhookCannon {
     if (newInterval > MAX_BACKOFF) {
       return lastInterval;
     }
-    return newInterval;
+    // RabbitMQ expects integer
+    return newInterval|0;
```

Comment:
I think you're missing a lint here to add some spacing in the expression. Either way, WDYT of the more explicit
```suggestion
    return Math.floor(newInterval);
```
or even rounding
```suggestion
    return Math.round(newInterval);
```
?

Reply:
I prefer `|0` because it always returns integer number. `Math.round(Infinity)` returns `Infinity`.